plugins/janus_streaming.c

```diff
@@ -3929,7 +4006,7 @@ static void *janus_streaming_handler(void *data) {
 static int janus_streaming_create_fd(int port, in_addr_t mcast, const janus_network_address *iface, const char *listenername, const char *medianame, const char *mountpointname) {
 	struct sockaddr_in address;
 	janus_network_address_string_buffer address_representation;
-	int fd = socket(AF_INET, SOCK_DGRAM, 0);
+	int fd = socket(AF_INET, SOCK_DGRAM, IPPROTO_UDP);
```

Comment:
I thought that was implicit considering the `SOCK_DGRAM` part, but that's fine, better safe than sorry :slightly_smiling_face: